coordinator/src/orderbook/trading.rs

```diff
@@ -224,7 +232,21 @@ async fn process_new_order(
                 }
                 Err(e) => {
                     tracing::warn!(%trader_id, order_id, "{e:#}");
-                    // TODO(holzeis): send push notification to user
+
+                    if let Some(user) = user::by_id(conn, trader_id.to_string())? {
```

Comment:
🔧 It would be nicer to use the push notification service if I wouldn't have to query the user here. I suggest moving the query to the notification_service and keep the api to only provide the `trader_id`.

Reply:
interesting idea; I'll experiment with such a thing in the follow-up!